pep-0581.rst

```diff
@@ -69,29 +74,33 @@ not currently available on Roundup / bpo.
   systematic filtering of emails.
 
 - Additional privacy, such as offering the user a choice to hide an
-  email address, while still allowing communication with the user through @-mentions.
+  email address, while still allowing communication with the user
+  through @-mentions.
+
 
 Issues with Roundup / bpo
 -------------------------
 
 - Less than five people maintain bpo. Some of them are core developers.
 
-- It is in Mercurial. Without any CI available, it puts heavy burden on the few
-  existing maintainers in terms of reviewing, testing, and applying patches.
+- The upstream Roundup code is in Mercurial. Without any CI available,
+  it puts heavy burden on the few existing maintainers in terms of
+  reviewing, testing, and applying patches.
 
-- At its current state, it is not equipped to accept lots of contributions from
-  people who aren't already familiar with the code base.
+  There is an open discussion about moving the source code of bpo to
+  GitHub [#]_. If the source code of bpo does move to GitHub, it will
+  become difficult to update patches from upstream. But as long as it
+  is in Mercurial, it is difficult to maintain and onboard new
+  contributors.
 
-- The upstream Roundup is in Mercurial. There is an open discussion about
-  moving the source code of bpo to GitHub [#]_. If the source code of
-  bpo does move to GitHub, it will become difficult to update patches from
-  upstream. But as long as it is in Mercurial, it is difficult to maintain
-  and onboard new contributors.
+- In its current state, the project is not equipped to accept lots of
+  contributions from people who aren't already familiar with the code
```

Comment:
I find the most complicated part of the installation process is installation of our Roundup fork. Once we get rid of our fork, it won't be harder than installing python.org.

I'd share my `INSTALL` document here, but unfortunately I left it in my computer in my hometown. Buy you can take a look at the Docker image at https://github.com/python/docker-bpo.